src/db/profile.sql

```diff
@@ -0,0 +1,10 @@
+-- Volunteer Profile Schema
+CREATE TABLE IF NOT EXISTS profile (
+    id TEXT PRIMARY KEY,               -- UUID stored as TEXT
```

Comment:
```suggestion
    id TEXT NOT NULL UNIQUE COLLATE BINARY,               -- UUID stored as TEXT
```

Same pedantic comment as before